docs/rules/no-constant-condition.md

```diff
@@ -41,6 +41,15 @@ do{
 var result = 0 ? a : b;
 ```
 
+```js
+/*eslint no-constant-condition: "error"*/
+
+if(typeof x){
+  doSomething();
```

Comment:
@platinumazure This is .md file, so it's not linted (well, not by ESLint anyways).